Formula/klee.rb

```diff
@@ -16,8 +16,8 @@ class Klee < Formula
 
   depends_on "cmake" => :build
   depends_on "gperftools"
+  depends_on "libpython-tabulate"
```

Comment:
I don't think this is right. This formula definitely uses `tabulate` by shelling out to it.

Reply:
Can you be more precise, I could not find where this is happening? I only found the `tools/klee-stats/klee-stats` file that imports the library.